libs/component-library/src/lib/theme/theme.ts

```diff
@@ -217,10 +219,10 @@ const buttonTheme = {
   },
   text: {
     types: {
-      defaults: { color: 'white' },
-      inverted: { color: 'greenBase' },
-      disabled: { color: 'border' },
-      pressed: { color: 'white' },
+      defaults: { color: 'white' } as ButtonThemeColorType,
```

Comment:
I was thinking about how to best type this. Ultimately as far as I can tell it can be any Predefined Restyle Function prop.

Reply:
We can expand the type to take addition props if we need. I had to add this typing around `color` to allow it be used with `button.tsx` to pass down the icon color